yarn.lock

```diff
@@ -5,14 +5,12 @@
 "@babel/code-frame@7.0.0-beta.51":
   version "7.0.0-beta.51"
   resolved "https://registry.yarnpkg.com/@babel/code-frame/-/code-frame-7.0.0-beta.51.tgz#bd71d9b192af978df915829d39d4094456439a0c"
-  integrity sha1-vXHZsZKvl435FYKdOdQJRFZDmgw=
```

Comment:
Please update your `yarn` version so all of these integrity entries aren't removed...

Reply:
```
[upgrade-broccoli-concat 6b14c85a] integrity checks!
 1 file changed, 995 insertions(+)
```
wow